### PR TITLE
chore: adds Buyer Guarantee seo meta tags 

### PR DESCRIPTION
**Pull request by ashleyjelks**
Screen Shot 2021-05-26 at 4 37 32 PM

https://artsyproduct.atlassian.net/browse/PURCHASE-2656

### CODE REVIEW

File: src/v2/Apps/BuyerGuarantee/Components/BuyerGuaranteeMeta.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { Meta, Title, Link } from "react-head"
+import { getENV } from "v2/Utils/getENV"
+import { cropped } from "v2/Utils/resized"
+
+export const BuyerGuaranteeMeta: React.FC = () => {
+  const title = "The Artsy Guarantee - Authenticity and Secure Payment"
+  const href = `${getENV("APP_URL")}/buyer-guarantee`
+  const description =
+    " Artsy is the safest place to buy the art you love. Every purchase made exclusively with Artsy’s secure checkout benefits from our full suite of buyer protections."
```
Comment (erikdstock) [inline src/v2/Apps/BuyerGuarantee/Components/BuyerGuaranteeMeta.tsx:10]: Is this leading space standard?

Comment (dzucconi) [inline src/v2/Apps/BuyerGuarantee/Components/BuyerGuaranteeMeta.tsx:10]: Should trim the leading space.

**Review by erikdstock — COMMENTED** [review]
Looks good here (though i'm partly reviewing to see if anyone more experienced with these tags & components has other thoughts)

**Review by dzucconi — APPROVED** [review]
👍

**Comment by dzucconi** [comment]
I wish there were a simpler way to debug these aside from deploying something and checking the Twitter/Facebook debuggers. There's a number of Chrome extensions but none of them work great (if anyone has any suggestions I'm all ears).

Twitter = https://cards-dev.twitter.com/validator
Facebook = https://developers.facebook.com/tools/debug/ (of course they require a Facebook login 😞 )

**Pull request merged**